src/attrs/__init__.pyi

```diff
@@ -0,0 +1,63 @@
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Mapping,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+)
+
+# Because we need to type our own stuff, we have to make everything from
+# attr explicitly public too.
+from attr import __author__ as __author__
```

Comment:
@euresti is there a better way to deal with this stuff maybe?

Reply:
would `__all__` help perhaps? some tools (flake8 afaik) look at it and will not warn about ‘unused imports’ for anything appearing there. not sure if similar things are in place for tools (mypy etc) looking at .pyi files.

Reply:
help with what exactly? i'd like to elide that whole block :)

the problem is that i need to have type hints for asdict/astuple and with that I can't use tacitly those from attr/__init__.pyi and have to import them here

Reply:
ack, i'm missing some context here. i was referring to the `x as x` repetition

Reply:
according to `attr/__init__.pyi` it’s necessary to make them public and in my experiments it still seems to hold true 🤔